apps/price_pusher/src/solana/solana.ts

```diff
@@ -125,6 +125,7 @@ export class SolanaPricePusher implements IPricePusher {
       this.logger.info({ signatures }, "updatePriceFeed successful");
     } catch (err: any) {
       this.logger.error(err, "updatePriceFeed failed");
+      return;
```

Comment:
this return doesn't seem to do anything

Reply:
lol it was there before and i removed it. added it back to minimize the changes.